bitspossessed/culturestake
# Expiring emails

Close #168 

The basic flow that this PR adds is that the expiring vote emails. When uploading via the schedule emails page, a link is sent to each voter that directs them to a "pre-vote page" - on the pre-vote page, they can enter their email. 

If there is a vote invitation associated with this email, they are sent another email, with a vote link that will expire in 15 minutes. If clicked, it sends them to the vote page. If clicked a second time, it no longer works.

If the user requests this link but doesn't follow through to voting, they can request a new, unique, expiring link. If they did finish voting, they can no longer request an expiring vote link

Steps to test this:

1) make a vote with all needed components (festival, artwork, artist, festival question, artwork question, etc)
2) use the schedule emails page to create an invitation for at least one email
3) navigate the link logged in the sent emails, and input the email in the form, which will log a new (expiring) email job
4) navigate to the link logged in the new expiring email job, where you should be able to successfully vote
